routes.nim

```diff
@@ -697,7 +697,7 @@ proc format*(
   let formattedText = readFile(filePath)
   let fullRange = Range(
     start: Position(line: 0, character: 0),
-    `end`: Position(line: int.high, character: int.high),
+    `end`: Position(line: cast[int](uint32.high), character: cast[int](uint32.high)),
```

Comment:
We should change the `Position` type, if you dont feel up to the task, please at least do `int(uint32.high)` rather than using `cast`

Reply:
Thank you, I will change this `int(`